src/Gelf/Transport/RetryTransportWrapper.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+
+namespace Gelf\Transport;
+
+use Gelf\MessageInterface as Message;
+
+
+class RetryTransportWrapper extends AbstractTransport
+{
+    /**
+     * @var AbstractTransport
+     */
+    private $transport;
+
+    /**
+     * RetryTransportWrapper constructor.
+     *
+     * @param AbstractTransport $transport
+     */
+    public function __construct(AbstractTransport $transport)
+    {
+        $this->transport = $transport;
+    }
+
+    /**
+     * Sends a Message over this transport.
+     *
+     * @param Message $message
+     *
+     * @return int calls function to send message
+     */
+    public function send(Message $message)
+    {
+        try {
+            return $this->transport->send($message);
+        } catch (\Exception $e) {
```

Comment:
To catch all exception, it's better to use the `Throwable` class, and can you import the class to provide a better overview on which out-of-namespace-classes are used?

```suggestion
        } catch (Throwable $e) {
```